frontend/src/pages/SearchPage.tsx

```diff
@@ -68,13 +70,19 @@ export const loader: Loader<SearchPageData> = async (request: Request) => {
   return { ...data, sort };
 };
 
-function pageUrl(query: string, page: number): string {
+function pageUrl(
+  query: string,
+  sort: VotesSearchSortOptions,
+  page: number,
+): string {
   const params = new URLSearchParams();
 
   if (query !== "") {
     params.set("q", query);
   }
 
+  params.set("sort", sort);
```

Comment:
Maybe only add the parameter when it’s not the default?

```suggestion
  if (sort !== DEFAULT_SORT) {
    params.set("sort", sort);
  }
```